### PR TITLE
Add a compatible Galaxy v1 api that ansible-galaxy cli can use

### PR DESCRIPTION
**Pull request by daviddavis**
fixes #3181
https://pulp.plan.io/issues/3181

### CODE REVIEW

File: pulp_ansible/app/serializers.py [hunk 1]
```diff
@@ -17,8 +17,13 @@ class AnsibleRoleSerializer(ContentSerializer):
         lookup_url_kwarg='role_pk',
     )
 
+    version_count = serializers.IntegerField(
+        source='versions.count',
+        read_only=True
+    )
```
Comment (daviddavis) [inline pulp_ansible/app/serializers.py]: This change wasn't necessary. It just made some of the debugging easier and I thought it was useful so I left it.

**Comment by bmbouter** [comment]
It needs conflict resolution. Also when I went to run the last step (w/ your modified galaxy client) I got this error:

```
(pulp) [vagrant@pulp3 ansible]$ ansible-galaxy install ansible.kubernetes-modules
- downloading role 'kubernetes-modules', owned by ansible
 [WARNING]: - ansible.kubernetes-modules was NOT installed successfully: Unable to compare role versions (ansible-kubernetes-modules-0.3.1-3, ansible-kubernetes-modules-0.3.1-5, ansible-
kubernetes-modules-0.4.0-7, ansible-kubernetes-modules-0.4.0-8, 0.4.0-alpha.0, ansible-kubernetes-modules-0.3.1-6, ansible-kubernetes-modules-0.3.1-4, v0.0.1, v0.3.1-5, v0.3.1-6, ansible-
kubernetes-modules-0.3.1-2) to determine the most recent version due to incompatible version formats. Please contact the role author to resolve versioning conflicts, or specify an
explicit role version to install.
```

When I did the manual install it was working.

**Comment by daviddavis** [comment]
I’ll rebase and look at the error when I get home. In the meantime, do you know how to specify a particular version when you install a role? That should work.

**Comment by daviddavis** [comment]
I figured out why you're hitting this error and I'm not. I'm guessing you're running against ansible against python 3? I have ansible 2.4.1 running with python 2.7. Here's the bug report about why this is happening in python 3:

https://github.com/ansible/ansible/issues/32301

It's ultimately caused by this python bug:

https://bugs.python.org/issue14894

If you try to install ansible-kubernetes from galaxy.ansible.com, it will also fail. It's not a problem with the code in this PR. I can try to find another example though for the README which doesn't fail. In the mean time, specifying a version should work:

```
ansible-galaxy install ansible.kubernetes-modules,v0.3.1-6
```

I've updated the README to use this command.

**Pull request merged**